pkg/agentfs/detection_test.go

```diff
@@ -0,0 +1,175 @@
+package agentfs
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestProjectDetection(t *testing.T) {
+	// Test UV project detection with the actual agent-starter-python example
+	uvProjectPath := "/Volumes/Code/workspaces/cloud-agents/livekit-examples/agent-starter-python"
```

Comment:
Obviously test like this won't work on anyone else's machine, but if we wanted to we could include some actual minimal examples in this repo and test them with `embed.FS`. I imagine Go's compiler is smart enough not to compile them into the binary if only referenced from test files? But I'm not sure.